packages/flutter/lib/src/widgets/icon.dart

```diff
@@ -93,15 +101,15 @@ class Icon extends StatelessWidget {
     final double iconSize = size ?? iconTheme.size;
 
     if (icon == null)
-      return new SizedBox(width: iconSize, height: iconSize);
+      return wrapWithSemantics(new SizedBox(width: iconSize, height: iconSize));
```

Comment:
This one doesn't need the ExcludeSemantics because there is never anything to exclude.